provider/management/commands/delete_expired_grant_tokens.py

```diff
@@ -52,27 +53,26 @@ def handle(self, *args, **options):
 
         delete_date = datetime.now()
         total_expired_tokens = Grant.objects.filter(expires__lte=delete_date).count()
+        ids = Grant.objects.all().aggregate(Min('id'), Max('id'))
+        min_id = ids['id__min']
+        max_id = ids['id__max']
 
         log.info(
-            "STARTED: Deleting %s expired grant tokens older than '%s' w/ chunk size of %s and %s secs between chunks",
-            total_expired_tokens, delete_date, chunk_size, sleep_between
+            "STARTED: Deleting %s expired grant tokens older than '%s' w/ chunk size of %s and %s secs between chunks. "
+            "Ids are %s - %s.",
+            total_expired_tokens, delete_date, chunk_size, sleep_between, min_id, max_id
             )
 
-        total_deletions = 0
-
         while True:
-            target_ids = Grant.objects.filter(expires__lte=delete_date).values_list('id', flat=True)[:chunk_size]
-            deletions_now = len(target_ids)
+            log.info("Deleting up to %s expired tokens, starting at %s", chunk_size, min_id+chunk_size)
```

Comment:
purely a wording thing - I read "Starting at " as id > min_id+chunk_size.  I have no idea what it should read "deleting up to %s tokens, id <= %s" to be literal?  
Mostly it confused me and made me reread the loop twice

Reply:
Cleaned that up to be "with id less than %s", which I just realized isn't quite true. I'll fix and push to PyPI once it clears testing.